File: Finish Guess the Number Game/Finish Guess the Number Game.js
class Guesser {
  constructor(number, lives) {
    this.number = number;
    this.lives = lives;
  }
  
  guess(n) {
    if(this.lives <=0){
      throw exception;
    }
    if(this.number==n){
      return n
    }else{
      this.lives-=1
      return false
    }
  }
}

File: Finish Guess the Number Game/Finish Guess the Number Game.py
class Guesser:
    def __init__(self, number, lives):
        self.number = number
        self.lives = lives
  
    def guess(self,n):
        if self.lives <= 0:
           raise Exception("Omae wa mo shindeiru")
        
        if self.number == n:
            return True
        else:
            self.lives -= 1
            return False
